app/celery/tasks.py

```diff
@@ -624,3 +624,28 @@ def process_returned_letters_list(notification_references):
             updated, updated_history, len(notification_references)
         )
     )
+
+
+@notify_celery.task(name='send-notify-no-reply')
+@statsd(namespace="tasks")
+def send_notify_no_reply(data):
```

Comment:
This is the Celery task that will be triggered by the Lambda (hooked up to SES). In reality, the Lambda will post a message to an SQS queue to trigger this task